app/styles/components/lts-table.css

```
@@ -0,0 +1,17 @@
+.lts-table {
+  border-spacing: 0;
+}
+
+.lts-table th {
+  text-align: left;
+}
+
+.lts-table td,
+.lts-table th {
+  padding: 0 1em 0.5em 0;
```

Can we use `rem` instead? I think `rem` is always relative to the base font size, whereas `em` is relative to the font size in a parent container (figuring out which container can be difficult, I think).

```suggestion
  padding: 0 1rem 0.5rem 0;
```

---

Used the spacing vars, didn't notice these until now.